velox/vector/tests/VectorTestBase.h

```diff
@@ -137,8 +137,10 @@ class VectorTestBase {
   }
 
   template <typename T>
-  FlatVectorPtr<EvalType<T>> makeFlatVector(const std::vector<T>& data) {
-    return vectorMaker_.flatVector<T>(data);
+  FlatVectorPtr<EvalType<T>> makeFlatVector(
+      const std::vector<T>& data,
+      const TypePtr& type = CppToType<T>::create()) {
```

Comment:
We can add CppToType<T>::create() for DecimalLong and DecimalShort? Thanks!

Reply:
This we cannot since Decimal types require precision and scale. We cannot infer these based on the cpp type alone which is simply the unscaled value portion.

Reply:
I see. Thanks and could you also comment this?

Reply:
done.